### PR TITLE
Treeval gxy fixes (Added RAPID_TOL entry point / updated resources)

### LINKED ISSUE #228: Update resources for GET_PAIRED_CONTACT_BED
### Description of the bug

Scale base.config resources for GET_PAIRED_CONTACT_BED from hic files size.

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### LINKED ISSUE #235: Create new entry point RAPID_TOL
### Description of feature

RAPID will no longer have tol-specific subworkflows (e.g. kmer - only used for spectra image creation) and will include juicer
RAPID_TOL will have kmer, but include custom changes for internal use (e.g. skipping juicer)

### LINKED ISSUE #236: Update resources: CRAM_FILTER_ALIGN_BWAMEM2_FIXMATE_SORT/CRAM_FILTER_MINIMAP2_FILTER5END_FIXMATE_SORT
### Description of feature

bwamem/minimap2
cpus - 16
mem - 30 x genome_size

### PR DESCRIPTION
**Pull request by weaglesBio**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/treeval/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Closes #228
Closes #235
Closes #236

### CODE REVIEW

File: modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf [hunk 1]
```diff
@@ -5,7 +5,7 @@ process CRAM_FILTER_MINIMAP2_FILTER5END_FIXMATE_SORT {
     container 'quay.io/sanger-tol/cramfilter_bwamem2_minimap2_samtools_perl:0.001-c1'
 
     input:
-    tuple val(meta), path(cramfile), path(cramindex), val(from), val(to), val(base), val(chunkid), val(rglines), val(ref)
+    tuple val(meta), path(cramfile), path(cramindex), val(from), val(to), val(base), val(chunkid), val(rglines), val(ref), path(reference)
```
Comment (DLBPointon) [inline modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf:8]: If these are the meta and reference. Should this not just be it's own input?

Comment (weaglesBio) [reply modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf:8]: I think val(ref) is possibly an unclear name, looking at hic_minimap2.nf it is actually the path to the minimap index.

Comment (yumisims) [reply modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf:8]: sorry guys, cramfilter modules take index as input not reference fasta.

Comment (weaglesBio) [reply modules/local/cram_filter_minimap2_filter5end_fixmate_sort.nf:8]: Shall I update the name then in the module?

**Pull request merged**